scripts/policy/frameworks/netcontrol/catch-and-release.zeek

```diff
@@ -270,11 +272,40 @@ event rule_added(r: Rule, p: PluginState, msg: string)
 	local bi = blocks[ip];
 
 	local log = populate_log_record(ip, bi, DROPPED);
+	log$plugin = p$plugin$name(p);
 	if ( msg != "" )
 		log$message = msg;
 	Log::write(CATCH_RELEASE, log);
 	}
 
+event rule_exists(r: Rule, p: PluginState, msg: string)
+	{
+	if ( !cr_check_rule(r) )
+		return;
+
+	local ip = subnet_to_addr(r$entity$ip);
+	local bi = blocks[ip];
+
+	local log = populate_log_record(ip, bi, INFO);
+	log$plugin = p$plugin$name(p);
```

Comment:
The only thought I had was to add a new `populate_named_log_record(p: PluginState, ...)` or so that calls the original `populate_log_record()` and does the enrichment, but not sure it's worth it.